tests/e2e/nft/grpc.go

```diff
@@ -337,42 +299,6 @@ func (s *IntegrationTestSuite) TestQueryNFTGRPC() {
 		expectErr bool
 		errorMsg  string
 	}{
-		{
-			name: "class id is invalid",
-			args: struct {
-				ClassID string
-				ID      string
-			}{
-				ClassID: "invalid_class_id",
-				ID:      ExpNFT.Id,
-			},
-			expectErr: true,
-			errorMsg:  "invalid class id",
-		},
-		{
-			name: "class id does not exist",
-			args: struct {
-				ClassID string
-				ID      string
-			}{
-				ClassID: "class",
-				ID:      ExpNFT.Id,
-			},
-			expectErr: true,
-			errorMsg:  "not found nft",
-		},
```

Comment:
nit: I think this one can be put back

Reply:
Just tested locally and it can be put back, but we can do a follow-up PR :)

Reply:
Merging then, I'll update the docs in a quick follow-up and re-add this test.